lib/cmd/common/git.js

```diff
@@ -55,7 +58,7 @@ function props(appId, cb) {
 // Git set properties
 // Note: although we're just setting scm related properties, the app update endpoint expects the full config object..
 function set(appId, k, v, cb) {
-  common.readApp(appId, function (err, cfg) {
+  common.readAppWithoutProject(appId, function (err, cfg) { 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)